### ISSUE TITLE
support 3d convolution with float64

### ISSUE THREAD

**Issue by Eshonar** [tensorflow/tensorflow#21295]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows 7
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
N/A
- **TensorFlow installed from (source or binary)**:
from pip install

- **TensorFlow version (use command below)**:
tensorflow 1.9.0

- **Python version**:
3.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
cuDNN 9.0 v7, CUDA9.1

- **GPU model and memory**:
GTX980Ti 6GB, Geforce Titan X 12GB

- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I'm working on some convolutional calculation on the Tensorflow and got some strange errors with the Double Precision numbers.
I needed to calculate those tensors in float64 because of the precision issues.

The conv3d works fine with me on float32, but on the Double precision, it only works on CPU and on the GPU it causes error.

So is there any way I can get access to this operation?

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

import tensorflow as tf
import numpy as np

with tf.device('/device:GPU:0'):

    a = tf.constant(2 * np.ones([1, 200, 200, 200, 9], dtype=np.float64), dtype=tf.float64)

    b = tf.constant(np.ones([3, 3, 3, 9, 9], dtype=np.float64), dtype=tf.float64)

    sess = tf.Session()

    init = tf.global_variables_initializer()

    sess.run(init)

    c = tf.nn.conv3d(a, b, strides=[1, 1, 1, 1, 1], padding='SAME')
    print(a)
    print(b)
    print(c)
    for i in range(1000):
        print(sess.run(c))

the output says

InvalidArgumentError (see above for traceback): Cannot assign a device for operation 'Conv3D': Could not satisfy explicit device specification '/device:GPU:0' because no supported kernel for GPU devices is available.
Registered kernels:
  device='CPU'; T in [DT_HALF]
  device='CPU'; T in [DT_FLOAT]
  device='CPU'; T in [DT_DOUBLE]
  device='GPU'; T in [DT_HALF]
  device='GPU'; T in [DT_FLOAT]

 [[Node: Conv3D = Conv3D[T=DT_DOUBLE, data_format="NDHWC", dilations=[1, 1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1, 1], _device="/device:GPU:0"](Const, Const_1)]]

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Mobile device

**Comment by facaiy** [comment]
Hi, I created PR #21356 to fix the issue.

**Comment by Eshonar** [comment]
Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
